spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/types/external.scala

```diff
@@ -134,10 +138,14 @@ object external {
     dbEntity.setAttribute("name", dbDefinition.name.toLowerCase)
     dbEntity.setAttribute(AtlasConstants.CLUSTER_NAME_ATTRIBUTE, cluster)
     dbEntity.setAttribute("description", dbDefinition.description)
-    dbEntity.setAttribute("location", dbDefinition.locationUri.toString)
-    dbEntity.setAttribute("parameters", dbDefinition.properties.asJava)
+
+    // FYI: commenting out attributes which are tied to Hive DB model definition
+//    dbEntity.setAttribute("location", dbDefinition.locationUri.toString)
```

Comment:
indentation?

Reply:
Thanks for pointing out. That was the default of IntelliJ behavior while commenting. Will fix.